grid/social-media/publish-post/maps/facebook.test.ts

```diff
@@ -0,0 +1,43 @@
+import { RecordingDefinitions } from '@superfaceai/testing';
+import { env } from 'process';
+
+import { publishPostTest } from './publish-post';
+
+const ACCESS_TOKEN_REDACTED_VALUE = 'PARAMS_pageAccessToken';
```

Comment:
It's nitpicking for sure but I'd consider mentioning that the value has been redacted so it's clear from the recodings, e.g.:

```suggestion
const ACCESS_TOKEN_REDACTED_VALUE = 'PAGE_ACCESS_TOKEN_VALUE_REDACTED';
```

Reply:
value changed